lib/src/sql/statements/define.rs

```diff
@@ -73,7 +74,11 @@ impl DefineStatement {
 			Self::Namespace(ref v) => v.compute(ctx, opt, txn, doc).await,
 			Self::Database(ref v) => v.compute(ctx, opt, txn, doc).await,
 			Self::Function(ref v) => v.compute(ctx, opt, txn, doc).await,
-			Self::Login(ref v) => v.compute(ctx, opt, txn, doc).await,
+			// DEFINE LOGIN has been deprecated. Use DEFINE USER instead
+			Self::Login(_) => Err(Error::Deprecated(
+				"DEFINE LOGIN has been deprecated. Use DEFINE USER instead".to_string(),
```

Comment:
Is it OK to say deprecated if it now errors? "No longer supported" perhaps? Alternatively would it make sense to treat it as `Self::User(_)` and log a warning saying its deprecated?

Reply:
I thought about simply treat it as Self::User, but I want people to migrate out of `LOGIN`, hence returning an error. Good point about the message, will change it.

Reply:
```suggestion
				"DEFINE LOGIN is no longer supported. Use DEFINE USER instead".to_string(),
```